packages/compass-components/src/components/document-list/element.tsx

```diff
@@ -466,6 +466,17 @@ export const HadronElement: React.FunctionComponent<{
         </div>
         <div className={elementDivider} role="presentation">
           :&nbsp;
+          {
+            /* TODO: figure out exact placement */
```

Comment:
How do we figure out exact placement? Is that pending w/ Claudia? If there is a ticket maybe we can link it.

Reply:
Yeah, I’ve opened a ticket and assigned it to her, added the reference here ;)